modules/programs/git.nix

```diff
@@ -310,6 +310,13 @@ in {
           '';
         };
 
+        package = mkOption {
+          type = types.package;
+          default = pkgs.delta;
+          defaultText = literalExpression "pkgs.delta";
+          description = "Package providing <command>delta</command>.";
+        };
```

Comment:
```suggestion
        package = mkPackageOption pkgs "delta" { };
```

Reply:
@ncfavier I refactored this section following your advice, thank you!